models/plane_cam/plane_cam.sdf.jinja

```diff
@@ -0,0 +1,983 @@
+        <gyroscopeBiasCorrelationTime>1000.0</gyroscopeBiasCorrelationTime>
+        <gyroscopeTurnOnBiasSigma>0.0087</gyroscopeTurnOnBiasSigma>
+        <accelerometerNoiseDensity>0.004</accelerometerNoiseDensity>
+        <accelerometerRandomWalk>0.006</accelerometerRandomWalk>
+        <accelerometerBiasCorrelationTime>300.0</accelerometerBiasCorrelationTime>
+        <accelerometerTurnOnBiasSigma>0.196</accelerometerTurnOnBiasSigma>
+      </plugin>
+      <frame name='geotagged_cam::__model__' attached_to='geotagged_cam::camera_link'>
+        <pose relative_to='__model__'>0.3 0 0.35 -3e-06 3e-06 3.14159</pose>
+      </frame>
+      <link name='geotagged_cam::camera_link'>
+        <inertial>
+          <pose>-0.041 0 -0.162 0 -0 0</pose>
+          <mass>0.1</mass>
+          <inertia>
+            <ixx>0.001</ixx>
+            <ixy>0</ixy>
+            <ixz>0</ixz>
+            <iyy>0.001</iyy>
+            <iyz>0</iyz>
+            <izz>0.001</izz>
+          </inertia>
+        </inertial>
+        <sensor name='camera_imu' type='imu'>
+          <always_on>1</always_on>
+          <imu/>
+        </sensor>
+        <sensor name='camera' type='camera'>
+          <pose>-0.051 0 -0.162 0 -0 3.14159</pose>
+          <camera>
+            <horizontal_fov>1</horizontal_fov>
+            <image>
+              <format>R8G8B8</format>
+              <width>1920</width>
+              <height>1080</height>
+            </image>
+            <clip>
+              <near>0.05</near>
+              <far>15000</far>
+            </clip>
+          </camera>
+          <always_on>1</always_on>
+          <update_rate>5</update_rate>
+          <visualize>1</visualize>
+          <plugin name='GstCameraPlugin' filename='libgazebo_gst_camera_plugin.so'>
+            <robotNamespace></robotNamespace>
+            <udpHost>127.0.0.1</udpHost>
+            <udpPort>{{ gst_udp_port }}</udpPort>
+          </plugin>
+          <plugin name='CameraManagerPlugin' filename='libgazebo_camera_manager_plugin.so'>
+            <robotNamespace/>
+            <interval>1</interval>
+            <width>3840</width>
+            <height>2160</height>
+            <maximum_zoom>8.0</maximum_zoom>
+            <video_uri>{{ video_uri }}</video_uri>
+            <system_id>{{ mavlink_id }}</system_id>
+            <cam_component_id>{{ cam_component_id }}</cam_component_id>
+            <mavlink_cam_udp_port>{{ mavlink_cam_udp_port }}</mavlink_cam_udp_port>
+          </plugin>
+        </sensor>
+        <pose relative_to='geotagged_cam::__model__'>0 0 0 0 -0 0</pose>
+        <self_collide>0</self_collide>
+        <enable_wind>0</enable_wind>
+        <kinematic>0</kinematic>
+      </link>
+      <joint name='geotagged_cam_joint' type='fixed'>
+        <parent>plane::base_link</parent>
+        <child>geotagged_cam::camera_link</child>
+      </joint>
+      <pose>1.01 0.98 0.83 0 -0 0</pose>
+    </model>
+</sdf>
```

Comment:
Please replace this file with plane_cam.sdf

Reply:
sir, typhoon drone support sdf.ninja, 

I created for the gstreamer support multiple plane camera.

Reply:
if you want to arrangement file, I can.

Reply:
What is the arrangement file?

Reply:
plane_cam.sdf.ninja file mr. @Jaeyoung-Lim .